website/docs/cloud-docs/agents/agents.mdx

```diff
@@ -0,0 +1,104 @@
+---
+page_title: Terraform Cloud Agents - Install and Run Agents - Terraform Cloud and Terraform Enterprise
+description: >-
+  Install, configure, and run agents on your own infrastructure.
+---
+
+# Install and Run Agents
+
+The agent software runs on your own infrastructure. The token you provide when starting the agent assigns it to a Terraform Cloud [agent pool](/cloud-docs/agents/agent-pools).
+
+## Operational Considerations
+
+The agent is distributed as a standalone binary which can be run on any supported system. By default, the agent runs in the foreground as a long-running process that will continuously poll for workloads from Terraform Cloud. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
```

Comment:
```suggestion
The agent is distributed as a standalone binary which can be run on any supported system. By default, the agent runs in the foreground as a long-running process that continuously polls for workloads from Terraform Cloud. We strongly recommend pairing the agent with a process supervisor to ensure that it is automatically restarted in case of an error.
```